CSS/styles.css

```diff
@@ -205,207 +202,186 @@ body .background-color {
         opacity: .7;
         padding-right: 20px;
     }
-    
-    #openNav {
```

Comment:
This was duplicated in the styles and didn't change any of the styling with the media query. So I removed it. Typically if you needed to change one thing in a style ID or Class, you could just add the value you needed changed and the rest of the styles would remain. In this case the styles were exactly the same, so there was no changes when the media query is being fired off.